app/packs/stylesheets/desfake/_variables.scss

```diff
@@ -211,12 +213,54 @@ $button-responsive-expanded: false;
 $card-background: $white;
 $card-font-color: $body-font-color;
 $card-divider-background: $primary-color;
-$card-border: 1px solid $primary-color;
+$card-border: 1px solid $light-gray;
 $card-shadow: none;
 $card-border-radius: $global-radius;
 $card-padding: $global-padding;
 $card-margin-bottom: $global-margin;
 
+// // 20. Forms
+// // ---------
+
+// $fieldset-border: 1px solid $primary-color;
```

Comment:
Like the other comment, many commented lines give me the impression they are here from a discarded or incomplete work.

Reply:
I added a comment to explain why I'm leaving those lines commented out :)